doc/theming/templates.rst

```diff
@@ -137,43 +137,21 @@ This new code does a few things:
    name as one of CKAN's default template files, will be used instead of the
    default file.
 
-Now, let's customize the CKAN front page. We first need to discover which
-template file CKAN uses to render the front page, so we can replace it.
-Set :ref:`debug` to ``true`` in your |development.ini| file::
-
-    [DEFAULT]
-
-    # WARNING: *THIS SETTING MUST BE SET TO FALSE ON A PRODUCTION ENVIRONMENT*
-    debug = true
-
-.. _debug footer:
```

Comment:
Let's keep the instructions to set debug mode. This setting gives helpful feedback when errors are created while editing.